contracts/handler/CompatibilityFallbackHandler.sol

```diff
@@ -150,7 +151,7 @@ contract CompatibilityFallbackHandler is TokenCallbackHandler, ISignatureValidat
             mstore(0x40, add(response, responseSize))
```

Comment:
Related to memory safety - are there alignment requirements for allocations in Solidity?